wcfsetup/install/files/acp/templates/packageStartInstall.tpl

```diff
@@ -6,16 +6,20 @@
 {include file='header'}
 
 <script data-relocate="true">
-	require(['Language', 'WoltLabSuite/Core/Acp/Ui/Package/Search'], function(Language, AcpUiPackageSearch) {
+	require([
+		"Language", "WoltLabSuite/Core/Acp/Ui/Package/QuickInstallation", "WoltLabSuite/Core/Acp/Ui/Package/Search"],
+		(Language, { setup }, AcpUiPackageSearch) => {
```

Comment:
Please import the QuickInstallation as `QuickInstallation` without destructuring and then call `QuickInstallation.setup()`, this makes the code easier to understand.